<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fm_principal</class>
 <widget class="QMainWindow" name="fm_principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cálculo de IMC</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>411</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 20px;</string>
    </property>
    <property name="text">
     <string>Calculo de IMC - Índice de massa corporal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_calcular">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>170</y>
      <width>281</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Calcular</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_fechar">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>520</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Fechar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_limpar">
    <property name="geometry">
     <rect>
      <x>317</x>
      <y>170</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Limpar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Para uma pessoa com a sua altura, a massa ideal está entre:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>409</y>
      <width>52</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Mínimo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>408</y>
      <width>52</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Máximo:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_massa_minimo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_massa_maximo">
    <property name="geometry">
     <rect>
      <x>264</x>
      <y>430</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_mensagem">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:20px;
border:1px solid;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>528</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Versão 1.0</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>50</y>
      <width>381</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Digite sua altura em metros:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_altura">
         <property name="styleSheet">
          <string notr="true">font-size: 20px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Digite sua massa em quilogramas:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_massa">
         <property name="styleSheet">
          <string notr="true">font-size: 20px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>210</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">font-size:20px;</string>
       </property>
       <property name="text">
        <string>Seu IMC: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdNumber_imc"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>416</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionSobre"/>
    <addaction name="actionSair"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSobre">
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
